src/main/java/org/jboss/netty/util/internal/ByteBufferUtil.java

```diff
@@ -31,10 +31,17 @@
         Method directBufferCleanerCleanX = null;
         boolean v;
         try {
-            directBufferCleanerX = Class.forName("java.nio.DirectByteBuffer").getMethod("cleaner");
+            ByteBuffer direct = ByteBuffer.allocateDirect(1);
```

Comment:
If it is possible to clean `direct`, then I do so in line 44: `directBufferCleanerCleanX.invoke(cleaner);`. If an exception is thrown, then there is no point in trying to clean the byte buffer anyway.